wiki/common-platforms/gazebo-simulation.md

```diff
@@ -0,0 +1,139 @@
+## Introduction
```

Comment:
This is a well-written entry. However, it does not conform to any of our coding standards and likely won't play well with Jekyll. Please [review the template](https://raw.githubusercontent.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/d595f7aac3f0000ce40d845113940171a4f8ff1a/_templates/template.md) and update your submission.

Reply:
I have made some changes in order to meet Jekyll coding standards.